src/storage/txn/commands/scan_stale_locks.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use crate::storage::txn::commands::{Command, CommandExt, TypedCommand};
+use tikv_util::collections::HashMap;
+use txn_types::{Key, TimeStamp};
+
+command! {
+    /// Scan locks for resolving according to `txn_status`.
+    ///
+    /// During the GC operation, this should be called to find out stale locks whose timestamp is
+    /// before safe point.
+    /// This should follow by a `ResolveLock`
+    ScanStaleLocks:
+        cmd_ty => (),
+        display => "kv::scan_stale_locks", (),
+        content => {
+            /// Maps lock_ts to commit_ts. If a transaction was rolled back, it is mapped to 0.
```

Comment:
These comments about `txn_status` exists both in `ScanStaleLocks` and `ResolveLock`.
I think maybe we can keep them in one of them and reference it on the other?

Reply:
> I think maybe we can keep them in one of them and reference it on the other?

Sounds good.